front/app/containers/Admin/projectFolders/containers/projects/ItemsInFolder.tsx

```diff
@@ -37,6 +37,11 @@ const ItemsInFolder = ({ projectFolderId }: Props) => {
     publicationStatusFilter: publicationStatuses,
   });
 
+  const canRemoveProjects = usePermission({
+    item: 'project_folder',
```

Comment:
Normally, you would pass the project folder object here, not just a string. Then the rule should decide whether for this particular folder the user has permission to manage projects. I understand in this case only admins can do this so probably fine to leave as is. Just wanted to point it out in case you can think of a use case where it's needed.

Reply:
Yes, that's what I thought too, but there were similar usages in the original code. As it is, the component doesn’t hold a reference to the folder object, so we’d have to rework its signature. Let’s leave it as it is for now because changing it would require a larger refactor.

Thanks for the review!